mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,253 @@
+            for i in sub_files:
+                if '.zip' in i:
+                    sub_dir = op.splitext(i)[0]
+                    z2 = zipfile.ZipFile(i)
+                    z2.extractall(limo_dir)
+                    z2.close()
+                    # continue decompressing if necessary
+                    sub_sub_files = [j for j in os.listdir(sub_dir) if '.zip' in j]
+                    for k in sub_sub_files:
+                        z3 = zipfile.ZipFile(os.path.join(sub_dir, k))
+                        z3.extractall(sub_dir)
+                        z3.close()
+                elif '.tar' in i:
+                    t = tarfile.open(os.path.join(limo_dir, i))
+                    t.extractall(limo_dir)
+            stdout.write(' [done]\n')
+
+    # update default if desired
+    _do_path_update(path, update_path, key, name)
+    return limo_dir
+
+
+@verbose
+def load_data(subject, path=None, interpolate=False, force_update=False, update_path=None,
+              url=None, verbose=None):  # noqa: D301
+    """Fetch subjects epochs data for the LIMO data set.
+    
+    Parameters
+    ----------
+    subject : int | str
+        Subject to use. Can be in the range from 1 to 18.
+        If string, must be 'S1', 'S2' etc.
+    path : str
+        Location of where to look for the LIMO data.  
+        If None, the environment variable or config parameter
+        ``MNE_DATASETS_LIMO_PATH`` is used. If it doesn't exist, the
+        "~/mne_data" directory is used.
+    interpolate : bool
+        Whether to interpolate missing channels.
+    force_update : bool
+        Force update of the dataset even if a local copy exists.
+    update_path : bool | None
+        If True, set the MNE_DATASETS_LIMO_PATH in mne-python
+        config to the given path. If None, the user is prompted.
+    url : str
+        The location from where the dataset should be downloaded, if not
+        found on drive.
+    %(verbose)s
+    
+    Returns
+    -------
+    epochs : MNE Epochs data structure
+        The epochs.
+    """  # noqa: E501
+    if url is None:
+        url = 'http://datashare.is.ed.ac.uk/download/DS_10283_2189.zip'
+
+    # set limo path, download and decompress files if not found
+    limo_path = data_path(url, path, force_update, update_path)
+
+    # subject in question
+    if isinstance(subject, int):
+        subj = 'S%i' % subject
+    elif isinstance(subject, str):
+        if not subject.startswith('S'):
+            raise ValueError('`subject` must start with `S`')
+        subj = subject
+
+    # -- 1) import .mat files
+    # epochs info
+    fname_info = op.join(limo_path, subj, 'LIMO.mat')
+    data_info = scipy.io.loadmat(fname_info)
+    # epochs data
+    fname_eeg = op.join(limo_path, subj, 'Yr.mat')
+    data = scipy.io.loadmat(fname_eeg)
+
+    # -- 2) get epochs information from structure
+    # sampling rate
+    sfreq = data_info['LIMO']['data'][0][0][0][0]['sampling_rate'][0][0]
```

Comment:
`data_info['LIMO']['data'][0][0][0][0]` is used 3+ times right? Maybe give it a name.